code/modules/cargo/orderconsole.dm

```diff
@@ -71,6 +65,9 @@
 	board.obj_flags |= EMAGGED
 	update_static_data(user)
 
+/obj/machinery/computer/cargo/on_construction()
+	. = ..()
+	circuit.configure_machine(src)
```

Comment:
This should apply to all computers, not just cargo since the circuit proc is made to be generic. (Move this behaviour to /obj/machinery/computer/on_construction())

Reply:
I applied it to /obj/machinery/on_construction, since that's where the circuit variable originates. This covers all computers, as well as other machines that have a circuit board (vendors, lathes, etc.).